bower.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "stencil-select",
+  "version": "0.1.0",
+  "description": "A styleable, accessible <select> component.",
+  "authors": [
+    "Mobify",
+    "Jeff Kamo <jeffkamo@mobify.com>",
+    "Kyle Peatt <kpeatt@gmail.com>",
+    "Ryan Frederick <ryan@ryanfrederick.com>"
```

Comment:
Should we assign authorship to just Mobify? Seems weird to have the three of us singled out over other contribs.